packages/react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -80,6 +80,14 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
             },
           },
         },
+      hasInput && {
```

Comment:
What is the purpose of these new styles?

Reply:
We are removing a check value.length check, So it will show X (close button) all the time, So fixing that using styles.

This is just like showing and hiding magnifying glass on focus (Used the similar styles).